src/app/gene-plot/gene-plot.ts

```diff
@@ -10,7 +10,7 @@ export class GenePlotModel {
 
   constructor(gene: Gene, rangeWidth: number, spacerLength: number = 150) {
     this.gene = gene;
-    this.spacerLength = spacerLength; // in px
+    this.spacerLength = spacerLength - (gene.transcripts.length >= 25 ? (gene.transcripts.length <= 50 ? gene.transcripts.length : 50) * 2.5 : 0); // in px
```

Comment:
A comment should be written explaining why we cap the spacerLength if there are too many transcripts

Reply:
This is alright for now, but a worthwhile improvement would be to track the amount of chromosomes a gene may be found on instead of the amount of transcripts - there may be genes with many transcripts on only two chromosomes, and reducing the spacerLength there would not be needed

Reply:
use `gene.chromosomes` and set the amount to something around 3, experiment with numbers